hs_core/templates/pages/my-resources.html

```diff
@@ -549,10 +558,10 @@ <h4 class="modal-title" id="delete-resource-title">Delete Resource</h4>
             }
         ];
         // TODO: fix the following references
-{#        $(document).ready(function () {#}
+        $(document).ready(function () {
 {#            start_resource_table();#}
 {#            $.fn.dataTable.ext.search.push(hs_resource_table_custom_search);#}
-{#        });#}
+        });
```

Comment:
Should these couple of commented lines be removed? If so, the whole ```$(document).ready()``` callback function can be removed, right?